helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: kyverno.all.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: webhooks
+    rules:
+    - alert: KyvernoWebhooksPresent
+      annotations:
+        description: 'Kyverno has no available replicas but webhooks are present.'
```

Comment:
It would be nice to include the actual webhook name in the description if that info is available in the alert

Reply:
We have it in the alert, I think the problem is that there are two different labels depending if it is validating or mutating, plus if you have 10 webhooks then the alert would be super crowded.